### DISCUSSION TITLE
Theme for Alacritty

### DISCUSSION THREAD

**Discussion by NeckBeardPrince** [diegoulloao/neofusion.nvim#6] [Ideas]
https://github.com/alacritty/alacritty

**Comment by NeckBeardPrince** [comment]
> https://github.com/alacritty/alacritty

https://github.com/alacritty/alacritty-theme/tree/master/themes

**Comment by diegoulloao** [comment]
Hey @NeckBeardPrince thanks for opening the topic! Actually I'm planning to port the iTerm2 theme to other well known terminals like: Kitty, Alacritty and WezTerm... 🎉

**Comment by diegoulloao** [comment]
@NeckBeardPrince https://github.com/diegoulloao/neofusion.alacritty/

**Comment by NeckBeardPrince** [comment]
Heck yeah, now I need to spend time configuring Alacritty only to get annoyed and keep using iterm2.

**Reply by diegoulloao** [reply]
I was planning to try it about 2 months ago, never did it... until now in just 5 minutes only for test the theme 😂
Same for kitty, I'm impressed with the speed. I'll try them calmly later.

**Reply by diegoulloao** [reply]
Same, still using `iTerm2` 😶‍🌫️
